.github/workflows/ci-publish-base-image.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         run: ./scripts/git-checks.sh
       - id: changes
         name: determine changes
-        uses: diem/actions/changes@78f868bd1170a959f0a9864e33df725690b45b9a
+        uses: diem/actions/changes@266156eaff7ba6e1253cf7019506c7b3702f73de
```

Comment:
we can also consider making releases in diem/actions

Reply:
Not sure it buys us all that much.   If we want to communicate backward compatibility outside diem it could be valuable.  I see us as always having to test on an upgrade.